_includes/code/graphql.additional.py

```diff
@@ -113,31 +114,35 @@ def test_gqlresponse(response_in, gqlresponse_in):
 }
 # END MultiplePropSorting GraphQL
 """
-gqlresponse = client.query.raw(gql_query)
-test_gqlresponse(response, gqlresponse)
+
+# TEST DISABLED - sandbox needed
+# gqlresponse = client.query.raw(gql_query)
+# test_gqlresponse(response, gqlresponse)
 
 
 # ===========================================
 # ===== Sorting by _additional property =====
 # ===========================================
+
 # START AdditionalPropSorting Python
-response = (
-    client.query
-    .get('JeopardyQuestion', ['question', 'answer'])
-    .with_additional('creationTimeUnix')
-    .with_sort({
-        # highlight-start
-        'path': ['_creationTimeUnix'],
-        'order': 'asc'
-        # highlight-end
-    })
-    .with_limit(3)
-    .do()
+from weaviate.collections.classes.grpc import Sort
+ 
+article=client.collections.get("JeopardyQuestion")
+response = article.query.fetch_objects(
+    return_metadata=wvc.MetadataQuery(creation_time=True),
+    sort=Sort(ascending=True, prop="_creationTimeUnix"),
+    limit=3
 )
 
-print(json.dumps(response, indent=2))
+for o in response.objects:
+    print( f"Answer: {o.properties['answer']}")
+    print( f"Points: {o.properties['points']}")
+    print( f"Question: {o.properties['question']}")
+
```

Comment:
This should probably show the metadata that we are sorting by - e.g.:

```python
    print(f"Answer: {o.properties['answer']}")
    print(f"Points: {o.properties['points']}")
    print(f"Question: {o.properties['question']}")
    print(f"Creation time: {o.metadata.creation_time}")
```

Reply:
@daveatweaviate did you see this comment?

Reply:
Updated